conf/blank.properties

```
@@ -61,4 +61,88 @@ jsonreport.sources=
+cf.adopter.name=
+
+# Possible values:
+# NONE - disable email
+# NORMAL - sends on email on each run
+# ONLY_NEW_MISSING_ATTACHMENTS_OR_ERRORS - sends on email when the missing attachments change or when there are errors
+#
+cf.email.mode=NONE
+# CSV of emails, required if cf.email.mode != NONE
+cf.email.recipents=
+
+## Required if cf.mode = REST
+# oeq.url=
+# oeq.oauth.client.id=
+# oeq.oauth.client.secret=
+
+# Only for REST cf.mode reports.
+cf.institution.shortname=
+
+# Only used for cf.mode = REST
+# Number of retries after receiving an exception (such as a connection timeout)
+# before the CheckFiles utility will exit.
+cf.max.tries=10
+
+# Only used for cf.mode = REST
+# For testing - number of tries to inject a 1 MS timeout limit before allowing
+# the request to be normal.  '0' will effectively disable this test.  Only fully
+# implemented for REST mode.
+cf.testing.timeout=0
+
+## Required if cf.mode = DB-*
+
+cf.db.url=
+cf.db.username=
+cf.db.password=
+cf.db.type=POSTGRE
+#Specify an absolute path to the Institutions directory.
+cf.filestore.directory=/my/filestore/dir/Institutions
+
+# Required for DB-* modes and must be a number greater than 1.
+cf.num.items.per.query=50
+
+# Not required.  Only for DB-* cf.mode reports.  CheckFiles will not run if it can't resolve all institutions shortnames
+# as filestore directories.  If an institution filestore handle was changed, the user needs to include the following line in the properties
+# files for each different institution filestore:
+cf.filestore.institution.handle.{originalInstitutionShortname}={newInstitutionShortname}
+
+## Optional Filters
+
+# Not required.  This is the UUID if cf.mode = REST, otherwise it's the DB ID of the collection.
+# Note:  when running CheckFiles in a DB-* mode, the tool displays all the institutions and collections -
+#  You can grab the needed ID from there.
+cf.filter.by.collection=
+
+# Not required.  This only applies to DB-* cf.mode reports.  This is the 'short_name of the institution (The 'Filestore' value in the server admin pages)'.
+cf.filter.by.institution=
+
+# Advanced filestore configuration.  Needs a line for each collection covered in the installation.
+# Example:  for an institution with ID 23, institution shortname CfTesting, a collection with
+# uuid 123456-7890-1234-567890, and adv filestore path /my/extra/data,  this line would look like:
+# cf.filestore.adv.filestore.23.123456-7890-1234-567890=/my/extra/data/Institutions/CfTesting/Attachments/123456-7890-1234-567890
+# Note:  Since this is a 'dynamic' property, it won't show up in the property list at runtime.
+cf.filestore.advanced.{inst-id}.{collection-uuid}=
+
+# Will compare the previous error report with the error report just ran.
+# Recommended to use in conjuction with email for an automated regular cron job
+# Values: true|false
+# Default:  false
+cf.compare.missing.atts.after.run=
```

newline at end of file

```suggestion
cf.compare.missing.atts.after.run=

```